src/core/player.cpp

```diff
@@ -292,7 +292,7 @@ void Player::RestartOrPrevious() {
 
 void Player::Stop(bool stop_after) {
   engine_->Stop(stop_after);
-  app_->playlist_manager()->active()->set_current_row(-1);
+  //app_->playlist_manager()->active()->set_current_row(-1);
```

Comment:
If you're certain about this, just delete it, don't leave commented out code lying around.